### DISCUSSION TITLE
Elasticsearch index recreate & update Not working on Production server

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#16505] [Q&A / Support]
Hello PIM team,

I have tried to recreate and update the Elasticsearch index using the command line. I checked the portal page, and after 2 hours, there was no update on the portal page, so try to kill the process and try the same command. getting an error of "ERROR: The command is already running."

As we are scheduled to go into production next week, I would greatly appreciate any guidance on setup validation or recommended best practices for the Pimcore production environment.

![image](https://github.com/pimcore/pimcore/assets/106297450/b0b8b250-e6a5-4bc5-9de2-63b936715a02)

Error:
![image](https://github.com/pimcore/pimcore/assets/106297450/ca241c9c-6ffa-4c21-a772-df7689ed6eba)

Please help us to resolve the issue.
@kingjia90
@fashxp 

Regards,
Arul

**Comment by yannic7** [comment]
https://pimcore.com/docs/platform/Data_Importer/Troubleshooting_FAQ/

This is for the Data Importer but it'll also fix your problem.
